creator-node/src/middlewares.js

```diff
@@ -56,47 +72,78 @@ async function syncLockMiddleware (req, res, next) {
  * Blocks writes if node is not the primary for audiusUser associated with wallet
  */
 async function ensurePrimaryMiddleware (req, res, next) {
-  const serviceRegistry = req.app.get('serviceRegistry')
   const start = Date.now()
+  let logPrefix = '[ensurePrimaryMiddleware]'
```

Comment:
I think we've done `|` as a separator here elsewhere

Reply:
yeah i've been using `[prefix]` mostly lately but its super inconsistent across